src/Microsoft.VisualStudio.ProjectSystem.Managed/ProjectSystem/Rules/PropertyPages/PackagePropertyPage.xaml

```diff
@@ -74,27 +74,27 @@
   <StringProperty Name="PackageIcon"
                   DisplayName="Icon file"
                   Description="The relative path from the package root to the icon file. In addition to setting this property you need to make sure that the file is included in the package. Image file size is limited to 1 MB. Supported file formats include JPEG and PNG. An image resolution of 128x128 is recommended."
-                  HelpUrl="https://docs.microsoft.com/nuget/reference/msbuild-targets#packing-an-icon-image-file" />
+                  HelpUrl="https://go.microsoft.com/fwlink/?linkid=2147134" />
 
   <StringProperty Name="RepositoryUrl"
                   DisplayName="Repository URL"
                   Description="Specifies the URL for the repository where the source code for the package resides and/or from which it's being built. For linking to the project page, use the 'Project URL' field, instead."
-                  HelpUrl="https://docs.microsoft.com/nuget/reference/nuspec#repository" />
+                  HelpUrl="https://go.microsoft.com/fwlink/?linkid=2147241" />
 
   <StringProperty Name="RepositoryType"
                   DisplayName="Repository type"
                   Description="Specifies the type of the repository. Default is 'git'."
-                  HelpUrl="https://docs.microsoft.com/nuget/reference/nuspec#repository" />
+                  HelpUrl="https://go.microsoft.com/fwlink/?linkid=2147241" />
```

Comment:
>HelpUrl="https://go.microsoft.com/fwlink/?linkid=2147241" /> [](start = 18, length = 60)

This is the same than the one above. Is this by design?

Reply:
Yes; the same section of documentation covers both.